core/ir/opnd.h

```diff
@@ -1535,6 +1537,15 @@ DR_API
 opnd_t
 opnd_create_immed_float(float f);
 
+DR_API
+/**
+ * Returns an immediate double operand with value \p d.
+ * The caller's code should use proc_save_fpstate() or be inside a
+ * clean call that has requested to preserve the floating-point state.
+ */
+opnd_t
+opnd_create_immed_double(double d);
```

Comment:
Please add to the changelist in release.dox this new API function.

Reply:
Done.